scheduler/src/cook/mesos/api.clj

```diff
@@ -1093,7 +1093,8 @@
   {:job [s/Uuid]})
 
 (def UpdateRetriesRequest
-  {(s/optional-key :jobs) [s/Uuid]
+  {(s/optional-key :job) s/Uuid
```

Comment:
@shamsimam you are right about the design... this is for backwards compatibility.  See the referenced issue https://github.com/twosigma/Cook/issues/272